setup.cfg

```diff
@@ -70,7 +67,7 @@ install_requires =
 	depinfo
 	requests
 	equilibrator_api <0.2;python_version>='3.5'
-python_requires = >=2.7,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*
+python_requires = >=3.5,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*
```

Comment:
```suggestion
python_requires = >=3.5
```

Reply:
This will prevent installation on 2.7, though.

Reply:
Yeah I was wondering if we needed that :D

Reply:
I think for now we should at least still allow installation on Python 2.7. So maybe just leave this line unchanged.